src/newViews/MyProfile/index.tsx

```diff
@@ -1,11 +1,269 @@
+                            sub
+                        >
+                            <TextInput
+                                name="firstName"
+                                disabled={disabled}
+                                onChange={onValueChange}
+                                value={value?.firstName}
+                                error={error?.fields?.firstName}
+                                label={_ts('myProfile', 'firstName')}
+                                placeholder={_ts('myProfile', 'firstName')}
+                                autoFocus
+                            />
+                            <TextInput
+                                name="lastName"
+                                disabled={disabled}
+                                onChange={onValueChange}
+                                value={value?.lastName}
+                                error={error?.fields?.lastName}
+                                label={_ts('myProfile', 'lastName')}
+                                placeholder={_ts('myProfile', 'lastName')}
+                            />
+                            <TextInput
+                                name="organization"
+                                disabled={disabled}
+                                onChange={onValueChange}
+                                value={value?.organization}
+                                error={error?.fields?.organization}
+                                label={_ts('myProfile', 'organization')}
+                                placeholder={_ts('myProfile', 'organization')}
+                            />
+                        </Container>
+                        <Container
+                            className={styles.preferences}
+                            sub
+                            heading={_ts('myProfile', 'preferences')}
+                        >
+                            <SelectInput
+                                name="language"
+                                disabled={disabled}
+                                onChange={onValueChange}
+                                value={value?.language}
+                                error={error?.fields?.language}
+                                label={_ts('myProfile', 'platformLanguage')}
+                                placeholder={_ts('myProfile', 'platformLanguage')}
+                                keySelector={langaugeKeySelector}
+                                labelSelector={languageLabelSelector}
+                                options={languageResponse?.results}
+                            />
+                            <List
+                                data={emailOptOutsOptions}
+                                renderer={Checkbox}
+                                keySelector={emailOptOutKeySelector}
+                                rendererParams={rowRendererParams}
+                            />
```

Comment:
Isn't this equivalent to a multiselect input? In that case we should use a multiselect input variant instead of handling these logic ourselves

Reply:
it is a multiselect input but we need to render them as checkboxes without opening the options dialog. Is it possible to do so ?

Reply:
I meant, we should have a MultiSelect variant which shows them outside the popup.
A input like ListSelection.
Shouldn't be too hard to do.

Reply:
Makes sense to have it.
@samshara , let's add a todo here and create a separate card for it and work on it.